DigitalLearningSolutions.Data.Migrations/202205061441_SeparateCentreEmailsTable.cs

```diff
@@ -37,6 +39,14 @@ public override void Down()
             Rename.Column("Email_deprecated").OnTable("AdminAccounts").To("Email");
             Rename.Column("Email_deprecated").OnTable("DelegateAccounts").To("Email");
 
+            Create.Index("IX_AdminAccounts_CentreId_Email").OnTable("AdminAccounts").OnColumn("CentreID")
+                .Ascending()
+                .OnColumn("Email").Ascending().WithOptions().NonClustered();
+
+            Create.Index("IX_DelegateAccounts_CentreId_Email").OnTable("DelegateAccounts").OnColumn("CentreID")
+                .Ascending()
+                .OnColumn("Email").Ascending().WithOptions().NonClustered();

```

Comment:
This isn't going to create the correct if not null indexes, you need the ones I created in the previous Up